simplekv/net/botostore.py

```diff
@@ -87,10 +88,41 @@ def _get_filename(self, key, filename):
             return k.get_contents_to_filename(filename)
 
     def _open(self, key):
+        from boto.s3.keyfile import KeyFile
+
+        class SimpleKeyFile(KeyFile):
+            def read(self, size=-1):
+                if self.closed:
+                    raise ValueError("I/O operation on closed file")
+                if size < 0:
+                    size = self.key.size - self.location
+                return KeyFile.read(self, size)
+
+            def seekable(self):
+                return True
+
+            def readable(self):
+                return True
+
+            def seek(self, offset, whence=os.SEEK_SET):
+                if self.closed:
+                    raise ValueError("I/O operation on closed file")
+                if whence == 0:
```

Comment:
`0` (and `1` in L.114, `2` in L.118) should be the constants from `os`, like `os.SEEK_SET` above.